docs/compress-options.md

```diff
@@ -69,6 +69,14 @@ Default: `false`
 
 Pretty print file sizes when logging.
 
+## createEmptyArchive
```

Comment:
Okay, nice. What about your request for changes? This one: https://github.com/gruntjs/grunt-contrib-compress/pull/186#pullrequestreview-6430123. I think I fixed all.